effekt/jvm/src/main/scala/effekt/Repl.scala

```diff
@@ -220,16 +220,16 @@ class Repl(driver: Driver) extends REPL[Tree, EffektConfig, EffektError] {
         module = extendedDefs
 
         // try to find the symbol for the def to print the type
-        d.ids.foreach(id => (context.symbolOption(id) match {
+        context.symbolOption(d.id) match {
           case Some(v: ValueSymbol) =>
-            Some(context.valueTypeOf(v))
+            Some(v, context.valueTypeOf(v))
           case Some(b: BlockSymbol) =>
-            Some(context.blockTypeOf(b))
+            Some(b, context.blockTypeOf(b))
           case t =>
```

Comment:
As said before, did you check this works with the REPL? I would guess it does not print anything. Also it should probably be foreach and not map.

Reply:
Prints nothing in the repl because d.id is empty. In case of ValDef we need d.binders.map(_.id)

Reply:
You could in principle match on d and special case if it is a ValDef.